scripts/build/redhat/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "etcher-electron",
+  "productName": "Etcher",
+  "genericName": "Etcher",
+  "homepage": "https://etcher.io/",
+  "icon": "assets/icon.png",
```

Comment:
That icon is the icon passed to the desktop application file(extension desktop) for the launcher icon. Gnome Software is supposed to use the same icon but I suppose its not getting the right size ? maybe we could have icons of other sizes ?